### DISCUSSION TITLE
"blacklisted by security settings"  --WTF?

### DISCUSSION THREAD

**Discussion by J-HS-L** [Tampermonkey/tampermonkey#1652] [Q&A]
I've never had this message before; what does it mean?
The only site (so far) I've seen it pop up on, is Twitter.com.

I thought maybe my 'Tweak New Twitter' userscript was the problem.
But disabling it made no difference...

I'll keep it disabled though I think, & use TNT's browser addon instead.

Thank-you.

**Comment by derjanb** [comment]
You can check this setting by configuring "Configuration Mode" to "Advanced" and looking for "Security > Blacklisted Pages".

By default `*://platform.twitter.com/widgets/*` is part of the list, because IIRC in the past this caused performance issues.

**Reply by NiteCyper** [reply]
Configuration Mode is at: chrome-extension://dhdgffkkebhmkfjojejmpbldmpobfkfo/options.html#nav=settings

Toolbar > pinned Extensions > right-click Tampermonkey icon > Options > Configuration Mode dropdown

**Comment by J-HS-L** [comment]
> At least one part of this page, is listed at the 'forbidden pages' setting!

So, what is the significance of it remaining on that blacklist;
Is it simply meant to be a reminder/warning, that... 
Blacklisted URLs carry a greater risk of perf, stability, & sec. issues, when used with user-scripts?

**Reply by J-HS-L** [reply]
> > At least one part of this page, is listed at the 'forbidden pages' setting!
> 
> So, what is the significance of it remaining on that blacklist; 
> Is it simply meant to be a reminder/warning, that... 
> Blacklisted URLs carry a greater risk of perf, stability, & sec. issues, when used with user-scripts?

@derjanb or Anyone??? Thanks again!

**Reply by J-HS-L** [reply]
Made the following change, but I still have the same warning on ...twitter.com?
![image](https://user-images.githubusercontent.com/49305869/205126527-0c2ca130-2329-4451-b419-fc8b64ce5b31.png)

**Comment by J-HS-L** [comment]
![image](https://user-images.githubusercontent.com/49305869/203247832-99ba1bde-173b-4014-ba7f-c6686e928439.png)

**Comment by NiteCyper** [comment]
Make it so clicking on "Blacklisted by security settings" shortcuts to unblacklisting it.
